src/sentry/runner/commands/run.py

```diff
@@ -777,6 +781,7 @@ def dev_consumer(consumer_names):
             synchronize_commit_group=None,
             synchronize_commit_log_topic=None,
             healthcheck_file_path=None,
+            dlq_topic=None,
```

Comment:
How come you're setting it to None and not the passed in value?

Reply:
I thought maybe on dev we want it to crash hard if they're sending invalid metric. But yeah I can set these to the actual dlq values. Do you know what the names in `consumer_names` are?

Reply:
Ah, didn't see this was the the devserver - this is fine then.